Add description field to test-defs (#46)
kyma-incubator/octopus: docs/crd-test-definition.md

| **spec.template** |    **YES**   | Describes the Pod that will be created. This field is of `PodTemplateSpec` type from the Kubernetes API. Find its detailed description [here](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.11/#podtemplatespec-v1-core)  |
| **spec.skip**     |    **NO**    | Indicates that a test should not be executed. The default value is `false`. This feature is not yet implemented. |
| **spec.disableConcurrency** | **NO** | Disallows running the given test concurrently. The default value is `false`. 
| **spec.timeout** | **NO** | Defines the maximal duration of a test, after which it is terminated and marked as **Failed**. This feature is not yet implemented.

> | **spec.description** | **NO** | Describes the test case in detail (e.g. scope, test scenario, edge cases, known limitations etc.).

Reviewer: ```suggestion
| **spec.description** | **NO** | Describes the details of the test case, such as the scope, the test scenario, edge cases, known limitations, etc.
```
